app/server/app/content/swagger/api-private.json

```diff
@@ -0,0 +1,1718 @@
+{
+  "openapi": "3.0.0",
+  "servers": [
+    {
+      "description": "Production",
+      "url": "https://owapps.app.cloud.gov/expertquery/api"
+    },
+    {
+      "description": "Staging",
+      "url": "https://owapps-stage.app.cloud.gov/expertquery/api"
+    },
+    {
+      "description": "Development",
+      "url": "https://owapps-dev.app.cloud.gov/expertquery/api"
+    },
+    {
+      "description": "Developer",
+      "url": " http://localhost:{port}/api",
+      "variables": {
+        "port": {
+          "default": 9090
+        }
+      }
+    }
+  ],
+  "info": {
+    "description": "",
+    "version": "1.0.0",
+    "title": "US EPA Expert Query",
+    "termsOfService": "https://edg.epa.gov/EPA_Data_License.html",
+    "contact": {
+      "email": ""
+    },
+    "license": {
+      "name": "Creative Commons Zero Public Domain Dedication",
+      "url": "https://creativecommons.org/publicdomain/zero/1.0/"
+    }
+  },
+  "paths": {
+    "/lookupFiles": {
```

Comment:
Can you also add in the `/health` and `health/etlGlossary` endpoints?

Reply:
Right, I knew something was missing.